HISTORY.rst

```diff
@@ -17,10 +17,12 @@ Unreleased Changes
   of near-zero results to be set to 0.0.
 * Fixed a malformed ``ValueError`` message when a corrupt raster was
   encountered in ``raster_calculator``.
-* Fixes an uncessary calculation that pre-fills slope raster GeoTIFFs with
+* Fixes an unnecessary calculation that pre-fills slope raster GeoTIFFs with
   nodata values.
 * Added a check to ``convolve_2d`` to verify that raster path/band tuples were
-  passed where expected and raise a useful Exception  if not.
```

Comment:
Ha! I saw this in a previous PR, but didn't have the heart to point out the double space then, haha.

Reply:
oh you definitely should, if you see an error and let it slide? now we're both at fault...

Reply:
We can discuss if it should be fixed later, but please don't ignore stuff like that even if it seems trivial.